src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/TelemetryClientService.kt

```diff
@@ -85,6 +85,7 @@ class TelemetryClientService(
       "visitStart" to formatDateTimeToString(visit.startTimestamp),
       "visitStatus" to visit.visitStatus.name,
       "applicationReference" to visit.applicationReference,
+      "hasPhoneNumber" to ((visit.visitContact.telephone != null).toString()),
```

Comment:
what happens when you do just
visit.visitContact.telephone?

Reply:
Need the true or false value as a string as we return value as MutableMap<String, String>.